src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -297,26 +308,32 @@ public StashDiffReport getPullRequestDiffReport(String project, String repositor
   /**
    * Reset all comments linked to a pull-request.
    */
-  public void resetComments(String project, String repository, String pullRequestId, StashDiffReport diffReport, StashUser sonarUser, StashClient stashClient) {
+  public void resetComments(String project, String repository, String pullRequestId,
+                            StashDiffReport diffReport, StashUser sonarUser, StashClient stashClient) {
     try {
+      // Let's call this "diffRep_loop"
       for (StashComment comment : diffReport.getComments()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS135)